doc/contribute.rst

```diff
@@ -0,0 +1,248 @@
+
+        $ make
+
+-  Documentation and high-coverage tests are necessary for enhancements
+   to be accepted.
+
+-  At least one paragraph of narrative documentation with links to
+   references in the literature (with PDF links when possible) and
+   the example.
+
+.. _coding-guidelines: http://scikit-learn.org/dev/developers/contributing.html#coding-guidelines
+.. _task list: https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments
+
+You can also check for common programming errors with the following
+tools:
+
+-  Code with good unittest coverage (at least 80%), check with::
+
+        $ pip install pytest pytest-cov
+        $ pytest -vsl .
+
+-  No flake8 warnings (which includes pep8 and pyflakes), check with::
+
+        $ pip install flake8
+        $ flake8 path/to/module.py
+
+Filing bugs
+===========
+We use Github issues to track all bugs and feature requests; feel free to
+open an issue if you have found a bug or wish to see a feature implemented.
+
+It is recommended to check that your issue complies with the
+following rules before submitting:
+
+-  Verify that your issue is not being currently addressed by other
+   issues_ or `pull requests`_.
+
+-  Please ensure all code snippets and error messages are formatted in
+   appropriate code blocks.
+   See `Creating and highlighting code blocks`_.
+
+-  Please include your operating system type and version number, as well
+   as your Python, scikit-learn, numpy, and scipy versions. This information
+   can be found by runnning the following code snippet::
+
+    python
+    import platform; print(platform.platform())
+    import sys; print("Python", sys.version)
+    import numpy; print("NumPy", numpy.__version__)
+    import scipy; print("SciPy", scipy.__version__)
+    import sklearn; print("Scikit-Learn", sklearn.__version__)
+   
+-  Please be specific about what estimators and/or functions are involved
+   and the shape of the data, as appropriate; please include a
+   reproducible_ code snippet
+   or link to a gist_. If an exception is raised,
+   please provide the traceback.
+
+.. _Creating and highlighting code blocks: https://help.github.com/articles/creating-and-highlighting-code-blocks
+.. _issues: https://github.com/paris-saclay-cds/ramp-board/issues
+.. _pull requests: https://github.com/paris-saclay-cds/ramp-board/pulls
+.. _reproducible: https://stackoverflow.com/help/mcve
+.. _gist: https://gist.github.com
+
+Documentation
+=============
+
+We are glad to accept any sort of documentation: function docstrings,
+reStructuredText documents (like this one), tutorials, etc.
+reStructuredText documents live in the source code repository under the
+doc/ directory.
+
+You can edit the documentation using any text editor and then generate
+the HTML output by typing ``make html`` from the doc/ directory.
+Alternatively, ``make`` can be used to quickly generate the
+documentation without the example gallery. The resulting HTML files will
```

Comment:
I don't think ramp-board has an example gallery, so you can probably remove this part.